amethyst_renderer/src/sprite.rs

```diff
@@ -179,6 +183,28 @@ impl SpriteSheetSet {
     }
 }
 
+/// Allows loading of sprite lists in RON format.
+#[derive(Clone, Deserialize, Serialize)]
+pub struct SpriteSheetFormat;
+
+impl SimpleFormat<SpriteSheet> for SpriteSheetFormat {
```

Comment:
I thought RonFormat could handle things like that? I'll check later

Reply:
Well this one also includes the texture ID at runtime which may come in handy, I guess.

Reply:
Although you could just load the Vec<Sprite> with RonFormat and add it to a new spritesheet, but maybe this is more straight forward?

Reply:
That's where examples come into play ;3